### PR TITLE
Added the aroma plugin

### PR DESCRIPTION
**Pull request by lorenzo**
https://github.com/gourmet/aroma

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,7 @@ Additional lists you might find useful:
 - [Enviroments plugin](https://github.com/josegonzalez/cakephp-environments) - Plugin to handle enviroments.
 - [Settings plugin](https://github.com/cakemanager/cakephp-settings) - A plugin to manage your settings via your database.
 - [Setup plugin](https://github.com/dereuromark/cakephp-setup) - Plugin to handle very basic environments.
+- [Aroma plugin](https://github.com/gourmet/aroma) - Databse based configuration
```
Comment (dereuromark) [inline README.md]: Typo Database and full stop missing.

Comment (dereuromark) [inline README.md]: Also order please ;)

Comment (lorenzo) [inline README.md]: didn't notice it was ordered

Comment (dereuromark) [inline README.md]: Was't my idea :P

**Pull request merged**